packages/odyssey-react-mui/src/theme/pxToRem.ts

```diff
@@ -0,0 +1,14 @@
+/*!
+ * Copyright (c) 2025-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+export const pxToRem = <Pixels extends number | string>(pixels: Pixels) =>
+  Number(pixels) / 14;
```

Comment:
I was using this function and ended up not using it, but this is something we need in Odyssey, so I left it.